### ISSUE TITLE
archive this repository

### ISSUE THREAD

**Issue by mio-19** [Akarin-project/StarLink#2]
@cakoyo
We do not need to maintain a fork of Spigot 1.15.2 and a fork of Paper 1.15.2 at the same time.

**Comment by WesleyVanNeck** [comment]
You mean spigot not craftbukkit

**Comment by WesleyVanNeck** [comment]
Also why if guys just focus on akarin and sotr just on starlink problem solved?

**Comment by sotr2333** [comment]
I agree with this, this project is unmaintainable, actually, it's already unable to be compiled on my local due to upstream API changes, I have to refactor the project structure but think it's better to dev a 1.15.2 version of akarin.
